tox.ini

```diff
@@ -1,20 +1,17 @@
 [tox]
-envlist = py27-{sqlite,pg}
+envlist = py27-{django16,django17,django18}
```

Comment:
ah didn't see this comment, it pointed at an outdated commit.

will fix this up, thanks.